pegasus/forms/volunteer_engineer_submission.rb

```diff
@@ -19,6 +19,7 @@ def self.normalize(data)
     result[:skype_s] = nil_if_empty stripped data[:skype_s]
     result[:helpouts_s] = nil_if_empty stripped data[:helpouts_s]
 
+    result[:age_18_plus_b] = required data[:age_18_plus_b]
```

Comment:
Added for completeness although this code does not appear to be called, due to the override in volunteer_engineer_submission_2015.rb